.github/workflows/maven.yaml

```diff
@@ -27,6 +27,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: 'Test for Sonar secret'
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: |
+          echo "SONAR_TOKEN_SET=$(test ${SONAR_TOKEN} && echo true)" >> $GITHUB_ENV
```

Comment:
I don't want to put SONAR_TOKEN into the environment as that would mean that all the steps see the SONAR_TOKEN secret.